### DISCUSSION TITLE
Adding Primary Key

### DISCUSSION THREAD

**Discussion by tbcomputerguy** [supabase/supabase#4057] [Questions]
In table view, I am trying to delete two null value rows from bad json post.  When I try it says you need a primary key.  I try to add the primary key to a UUID field and i get this response : Column ID of relation "my table" contains null values.  I try to remove or change the value from null and I get a "You need a primary column"  I am in a loop.

Please help

Update:  I did a delete from postman, and deleted all rows.  Then did a post from a json file I had and all is good

**Comment by TheNextCEO** [comment]
try this 
`ALTER TABLE <table_name>
ADD PRIMARY KEY (<row>);`
